internal_ws/ykpack/src/types.rs

```diff
@@ -446,6 +449,8 @@ impl IPlace {
                     ty: new_ty,
                 }
             }
+            // are these unreachable or todos? if the former, the strings are unnecessary (and just
+            // lead to a bit of pointless binary bloat)
```

Comment:
I'm not sure if these cases can arise or not.

The strings come from a time before panic would give a line number.

Reply:
Fixed in 396b940.